__tests__/app.test.js

```diff
@@ -125,4 +125,44 @@ describe('App', ()=>{
         )})
         })
     })
+    describe('GET /api/articles/:article_id/comments', () => {
+        test('GET 200: should receive an array of all comment objects associated with an article_id', () => {
+            return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.comments).toBeInstanceOf(Array)
+                body.comments.forEach((comment) => {
+                    expect(comment).toMatchObject({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        article_id: expect.any(Number)
```

Comment:
Am I expecting any article id? I'm looking for comments on article 1. I'd be super confused for comments from article 4